src/content/decentralization/index.md

```diff
@@ -18,57 +18,144 @@ summaryPoints:
+Peer-to-peer systems are a classic internet example. Because users double as servers (providing computer power while using the services) these protocols are decentralized by default. Examples include BitTorrent and Skype. In fact, much of the underlying technology of the internet is based on [end-to-end networking](https://en.wikipedia.org/wiki/End-to-end_principle), an idea strongly related to peer-to-peer networks.
 
-You probably know multiple centralized payment-processing systems — Visa, Zelle, Western Union, Paypal, whatever. But do you know how these systems work?
+You could say that it still works this way, but that some of the "ends" have become extremely powerful and popular. In other words, rather than talking directly with the devices of our friends, family members, and random internet people (something that is completely possible through direct peer-to-peer or highly decentralized architectures), we all talk to social media servers.
 
-You would have to learn a lot to be able to *prove* that these systems were trustworthy. Of course, the real barrier to detailed knowledge is social. Apart from learning the technical skills, you would need to secure a management position in the company to get access to the details.
+## Decentralization in Ethereum {#ethereum-decentralized}
 
-When you look at it that way, the costs don't add up. Assuming you have zero technical knowledge, and that you need to prove that you could trust each payment processing system, here's the minimum you would need to invest:
+First things first: Nobody owns Ethereum. The [Ethereum Foundation](https://ethereum.foundation) provides support, but so do tons of other individuals and organizations.
 
-- Visa: Dedicating your career to becoming an engineering lead at Visa. 
-- Ethereum: An hour or two of [studying](/developers/docs/smart-contracts/languages/) [publicly](/developers/docs/) [available](/developers/tutorials/) [materials](/developers/learning-tools/) every day for 3-6 months.
+Building on that, Ethereum is decentralized in many other important ways. People operate nodes across the world, there are many different client software options, and it interoperates (often via third-party [decentralized apps](https://ethereum.org/en/dapps/)) with a lot of other decentralized protocols. It is built to resist spammers, to provide clear consensus, and to be unprofitable (or at least proportionately expensive) to attack. 
 
-Bonus: If you know anything at all about JavaScript or Python, you already have a huge head start.
+Ethereum is also *centralized* in a very important way — there is an agreed-upon set of rules (in the form of computer code) that every node should follow. This allows the system to operate predictably. However, it bears mentioning that these rules are upgraded and maintained in a decentralized way.
+
+### Smart contracts {#smart-contracts}
```

Comment:
Describing smart contracts in this way at this point in the page feels out of place. What's the purpose behind bringing them up here?

Reply:
Yeah.... this section did not get my point across, thanks for pointing that out. I expanded on what I was trying to say in the most recent commit.